CLIENT_PATH is the only environment when executing dhclient-script,
without this patch, dhclient-script won't run properly because it
invokes ifconfig and route

7/28/2010 - qhe

diff -ru dhcp-4.1.1-P1.orig//client/Makefile.am dhcp-4.1.1-P1/client/Makefile.am
--- dhcp-4.1.1-P1.orig//client/Makefile.am	2010-07-29 13:20:05.000000000 +0800
+++ dhcp-4.1.1-P1/client/Makefile.am	2010-07-29 13:28:14.000000000 +0800
@@ -10,9 +10,9 @@
 EXTRA_DIST = $(man_MANS)
 
 dhclient.o: dhclient.c
-	$(COMPILE) -DCLIENT_PATH='"$(sbindir)"' \
+	$(COMPILE) -DCLIENT_PATH='"PATH=$(sbindir):$(base_sbindir):$(bindir):$(base_bindir)"' \
 		   -DLOCALSTATEDIR='"$(localstatedir)"' -c dhclient.c
 
 dhc6.o: dhc6.c
-	$(COMPILE) -DCLIENT_PATH='"$(sbindir)"' \
+	$(COMPILE) -DCLIENT_PATH='"PATH=$(sbindir):$(base_sbindir):$(bindir):$(base_bindir)"' \
 		   -DLOCALSTATEDIR='"$(localstatedir)"' -c dhc6.c
